### DISCUSSION TITLE
Error occurs when running pip install fastapi (venv)

### DISCUSSION THREAD

**Discussion by vulcanidic95** [fastapi/fastapi#11173] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
$ python -m venv .venv
$ source ./venv/bin/activate
(.venv) radian@server14:~/workspace/temp/.venv/bin$ pip install fastapi
```

### Description

(.venv) radian@server14:~/workspace/temp/.venv/bin$ pip install fastapi
Looking in indexes: https://pypi.org/simple, https://test.pypi.org/simple/
Collecting fastapi
  Downloading https://test-files.pythonhosted.org/packages/fc/3f/1418252ee07b6bc3d1c8da2c5a2b24fd98200c1e534ff517f5006b9ba7bd/FASTAPI-1.0.tar.gz (2.5 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-p814x2m2/fastapi_a41d772c07094496b301dc079b413def/setup.py", line 3, in <module>
          with open('DESCRIPTION.txt') as file:
      FileNotFoundError: [Errno 2] No such file or directory: 'DESCRIPTION.txt'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

[notice] A new release of pip is available: 23.0.1 -> 24.0
[notice] To update, run: pip install --upgrade pip

### Operating System

Linux

### Operating System Details

Ubuntu 20.04 

### FastAPI Version

install error

### Pydantic Version

install error

### Python Version

Python 3.10.13

**Comment by vulcanidic95** [comment]
The commands below work well.
$ pip install fastapi==0.1.16
$ pip install fastapi==0.109.2
$ pip install fastapi==0.109.1
$ pip install fastapi==0.109.0
$ pip install fastapi==0.108.0

The command below does not work.
$ pip install fastapi

**Comment by GodMoonGoodman** [comment, marked as answer]
try ```pip install fastapi -i https://pypi.org/simple```
not
try ```pip install fastapi -i https://test.pypi.org/simple```

**Reply by GodMoonGoodman** [reply]
Package on test.pypi.org is broken 

```
~/Downloads/FASTAPI-1.0 > ls -al
total 40
drwxrwxrwx@   9 supemroon  staff   288  9 29  2022 .
drwx------@ 137 supemroon  staff  4384  2 21 10:03 ..
drwxrwxrwx@   7 supemroon  staff   224  9 29  2022 FASTAPI.egg-info
-rw-rw-rw-@   1 supemroon  staff  1076  9 29  2022 LICENSE.txt
-rw-rw-rw-@   1 supemroon  staff   328  9 29  2022 PKG-INFO
-rw-rw-rw-@   1 supemroon  staff   537  9 29  2022 README.md
-rw-rw-rw-@   1 supemroon  staff    42  9 29  2022 setup.cfg
-rw-rw-rw-@   1 supemroon  staff   602  9 29  2022 setup.py
drwxrwxrwx@   4 supemroon  staff   128  9 29  2022 venv
```


```
# setup.py

from setuptools import setup

with open('DESCRIPTION.txt') as file:
    long_description = file.read()

with open('requirements.txt') as f:
    required = f.read().splitlines()

setup(name='FASTAPI',
      version='1.0',
      description='A small api that uses fastapi-users',
      long_description=long_description,
      url = "https://github.com/ladejavu/python-dev/tree/docker",
      author='sundus_ladejavu',
      author_email='sundus.khalid@ladejavu.com',
      license='MIT',
      packages=['venv'],
      install_requires=required,
      keywords='fastapi'
      )
```

**Comment by hairem** [comment]
I get the same error when I use 'pip install "fastapi[standard]"' like the walkthough says but the fix here doesn't seem to let me run the walkthrough server using "fastapi dev main.py". I also get the error  metadata-generation-failed, when I tried to install it using pip install fastapi.

**Reply by wimpywarlord** [reply]
+1
